pages/resilience.js

```diff
@@ -1,24 +1,199 @@
+          "listen to your signs of distress, such as being stressed, anxious or depressed",
+          "seek assistance or tell someone who listens",
+        ]
+      },
+      {
+        title: "Identifying and prioritizing your physical health",
+        points: [
+          "get enough sleep",
+          "have a balanced and nutritious diet",
+          "have regular check-ups with your health provider(s)",
+          "Engaging in exercises that train your mind and help with mood",
+          "meditation or yoga",
+        ]
+      },
+      {
+        title: "Exercising your body through physical activities",
+        points: [
+          "running, walking, going to the gym",
+          "enjoying hobbies",
+        ]
+      },
+    ]
+  },
+  copingPursuits: {
+    foreground: "#95A9EF",
+    background: "white",
+    width: 450,
+    height: 250,
+    margin: 10,
+    boxes: [
+      {
+        title: "Identifying your strengths",
+        points: [
+          "what are you good at?",
+          "how can these strengths help you with your current challenges and activities?",
+          "relate your strengths to your goals",
+        ]
+      },
+      {
+        title: "Find meaning in life’s challenges",
+        points: [
+          "approaching challenges as new experiences to learn and grow",
+          "ask yourself what motivates you in different circumstances",
+        ]
+      },
+      {
+        title: "Explore new opportunities",
+        points: [
+          "challenge yourself to try new things, big or small",
+        ]
+      },
+      {
+        title: "Everyone experiences failures",
+        points: [
+          "do not let failures define your story or your worth as a person",
+          "figure out what you can learn from these experiences",
+          "examine what opportunities or change can come from these failures",
+        ]
+      },
+    ]
+  },
+  externalResilience: {
+    foreground: "lightblue",
+    background: "#7AF0F7",
+    width: 500,
+    height: 200,
+    margin: 0,
+    boxes: [
+      {
+        title: null,
+        points: [
+          "managing your finances to allow you to meet your needs",
+          "budgeting",
+          "engage in financial planning",
+          "accessing social services that assists with finances and education",
+          "identifying, or creating an identity",
+          "find family, friends, or communities in which you feel supported",
+        ]
+      },
+      {
+        title: null,
+        points: [
+          "make decisions, large or small, that help you feel in control of your life",
+          "access support services and resources available to you",
+          "know your rights to social, legal, and health services in your community",
+          "know where to seek help should you require these services e.g., web search, local government websites",
+          "ensure you can physically access them if needed",
+        ]
+      },
+    ]
+  },
+}
+
```

Comment:
it is fine if this is long

Some Ideas for another time
- create classes and instances to hold yr data. I know in C++ there is the data struct type which is such a clean pattern

Reply:
we can also pull from api endpoints in nextjs next time. We'll leave it for now and get back to it at another time imo.